File: Amigo.Tenant.UI.Web/src/app/amigotenant/expense/dto/expense-search-request.ts
export class SearchBase{
        public pageSize: number;
        public page: number;
}

export class ExpenseSearchRequest extends SearchBase
{
        public houseTypeId : number | null;
        public propertyName : string | null;
        public expenseDateFrom : Date;
        public expenseDateTo : Date;
        public periodId : number | null;
        public referenceNo : string | null;
        public totalAmountFrom : number | null;
        public totalAmountTo : number | null;
        public paymentTypeId : number | null;
        public conceptName : string | null;
        public businessPartnerId : number | null;
        public fileName : string | null;
        public tenantFullName: string | null;
}

File: Amigo.Tenant.Application.DTOs/Requests/Expense/ExpenseSearchRequest.cs
using Amigo.Tenant.Application.DTOs.Requests.Common;
using System;
using System.Collections.Generic;

namespace Amigo.Tenant.Application.DTOs.Requests.Expense
{
    public class ExpenseSearchRequest : PagedRequest
    {
        public DateTime? ExpenseDateFrom { get; set; }
        public DateTime? ExpenseDateTo { get; set; }
        public int? PaymentTypeId { get; set; }
        public int? HouseTypeId { get; set; }
        public int? PeriodId { get; set; }
        public string ReferenceNo { get; set; }
        public string Remark { get; set; }
        public decimal? TotalAmountFrom { get; set; }
        public decimal? TotalAmountTo { get; set; }
        public int? BusinessPartnerId { get; set; }
        public string FileName { get; set; }
        public string PropertyName { get; set; }
        public string ConceptName { get; set; }


    }
}
